packages/@okta/vuepress-site/docs/guides/sign-into-mobile-app-redirect/main/ios/opensignin.md

```diff
@@ -0,0 +1,26 @@
+Update the `signIn` function of `ContentView.swift` to sign the user in:
```

Comment:
Sign the user in by updating the `signIn` function of `ContentView.swift`:

Reply:
updated